tests/test_process.py

```diff
@@ -108,6 +111,157 @@ def test_load_rain_file(rain_data_file):
     ]
 
 
+def test_load_rain_file_flanders(rain_data_file_flanders):
+
+    """Valid rainfall data should be parsed to rain DataFrame"""
+    rainfall_data = load_rain_file(rain_data_file_flanders, load_rain_file_flanders)
+
+    assert isinstance(rainfall_data, pd.DataFrame)
+
+    assert list(rainfall_data.columns) == [
```

Comment:
you can drop this assert as this is handled by https://github.com/watem-sedem/rfactor/blob/master/tests/test_process.py#L82. Tag and year is self generated, so this doesnt need to be tested https://github.com/watem-sedem/rfactor/blob/68e68a454af177ae5445be8d50a7f2a23545676f/src/rfactor/process.py#L148